graphql/context_field.go

```diff
@@ -36,8 +36,8 @@ type FieldContext struct {
 	//
 	//	srv.AroundFields(func(ctx context.Context, next graphql.Resolver) (interface{}, error) {
 	//		fc := graphql.GetFieldContext(ctx)
-	//		op := graphql.GetOperationContext(ctx)
-	//		collected := graphql.CollectFields(opCtx, fc.Field.Selections, []string{"User"})
+	//		rc := graphql.GetOperationContext(ctx)
```

Comment:
Actually, there are some variations in the naming of the variable returned from `graphql.GetOperationContext()` (e.g., `rc`, `opCtx`, `rctx`), but `rc` is the most common.